math/wide_integer/uintwide_t.h

```diff
@@ -1222,7 +1258,7 @@
               && (right_shift_amount_v < unsinged_fast_type(std::numeric_limits<UnsignedIntegralType>::digits)));
              ++index_u)
       {
-        *(values.begin() + size_t(index_u)) = limb_type(v >> (int) right_shift_amount_v);
```

Comment:
Did you change this in response to _hicpp-signed-bitwise_? FYI, I don't agree with that check. I would not be at all concerned if you left it turned off in .clang-tidy.